src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -21,6 +21,35 @@ Public Class ParseDirectives
         ]]>)
     End Sub
 
+    Public Sub ParseShebangDirective()
+        ParseAndVerify("#!", TestOptions.Script)
```

Comment:
I may just be missing them, but I don't see any tests of shebang in regular code.